client/src/components/Snackbar/index.jsx

```diff
@@ -1,17 +1,25 @@
 /* eslint-disable react/require-default-props */
 import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import CheckCircleIcon from '@material-ui/icons/CheckCircle';
-import ErrorIcon from '@material-ui/icons/Error';
-import InfoIcon from '@material-ui/icons/Info';
-import CloseIcon from '@material-ui/icons/Close';
-import { amber, green } from '@material-ui/core/colors';
-import IconButton from '@material-ui/core/IconButton';
+import { makeStyles } from '@material-ui/core/styles';
+import { IconButton } from '@material-ui/core';
 import Snackbar from '@material-ui/core/Snackbar';
 import SnackbarContent from '@material-ui/core/SnackbarContent';
-import WarningIcon from '@material-ui/icons/Warning';
-import { makeStyles } from '@material-ui/core/styles';
+import {
+  Error as ErrorIcon,
+  Info as InfoIcon,
+  Close as CloseIcon,
+  CheckCircle as CheckCircleIcon,
+  Warning as WarningIcon,
+} from '@material-ui/icons';
+import { amber, green } from '@material-ui/core/colors';
```

Comment:
need to sort this so we don't have many variants of green on the app, but we can go over this in a UI/UX check so okay for now

Reply:
Can we make an issue for this?

Reply:
Added issue #34